cmd/kubeops/internal/handler/handler_test.go

```diff
@@ -117,10 +118,83 @@ func TestActions(t *testing.T) {
 			},
 			ResponseBody: "",
 		},
+		{
+			// Scenario params
+			Description: "Delete a simple release",
+			ExistingReleases: []release.Release{
+				createRelease("foobarchart", "foobar", "default", 1, release.StatusDeployed),
+			},
+			DisableAuth: true,
+			// Request params
+			RequestBody:  "",
+			RequestQuery: "",
+			Action:       "delete",
+			Params:       map[string]string{"namespace": "default", "releaseName": "foobar"},
+			// Expected result
+			StatusCode: 200,
+			RemainingReleases: []release.Release{
+				createRelease("foobarchart", "foobar", "default", 1, release.StatusUninstalled),
+			},
+			ResponseBody: "",
+		},
+		{
+			// Scenario params
+			Description: "Delete and purge a simple release with purge=true",
+			ExistingReleases: []release.Release{
+				createRelease("foobarchart", "foobar", "default", 1, release.StatusDeployed),
+			},
+			DisableAuth: true,
+			// Request params
+			RequestBody:  "",
+			RequestQuery: "?purge=true",
+			Action:       "delete",
+			Params:       map[string]string{"namespace": "default", "releaseName": "foobar"},
+			// Expected result
+			StatusCode:        200,
+			RemainingReleases: []release.Release{},
+			ResponseBody:      "",
+		},
+		{
+			// Scenario params
+			Description: "Delete and purge a simple release with purge=1",
+			ExistingReleases: []release.Release{
+				createRelease("foobarchart", "foobar", "default", 1, release.StatusDeployed),
+			},
+			DisableAuth: true,
+			// Request params
+			RequestBody:  "",
+			RequestQuery: "?purge=1",
+			Action:       "delete",
+			Params:       map[string]string{"namespace": "default", "releaseName": "foobar"},
+			// Expected result
+			StatusCode:        200,
+			RemainingReleases: []release.Release{},
+			ResponseBody:      "",
+		},
+		{
+			// Scenario params
+			Description:      "Delete a missing release",
+			ExistingReleases: []release.Release{},
+			DisableAuth:      true,
+			Skip:             true,
+			// Request params
+			RequestBody:  "",
+			RequestQuery: "",
+			Action:       "delete",
+			Params:       map[string]string{"namespace": "default", "releaseName": "foobar"},
+			// Expected result
+			StatusCode:        404,
+			RemainingReleases: []release.Release{},
+			ResponseBody:      "",
+		},
 	}
 
 	for _, test := range tests {
 		t.Run(test.Description, func(t *testing.T) {
+			// TODO Remove this `if` statement after the memory driver bug is fixed
```

Comment:
Thanks @latiif . Can you reference the actual Issue/PR (PR in this case) to make it easier to audit later? Thanks. (both in this PR and the previous test one)